snuba/subscriptions/executor.py

```diff
@@ -23,8 +23,9 @@ def __init__(self, dataset: Dataset, executor_pool: ThreadPoolExecutor):
         self.__executor_pool = executor_pool
 
     def execute(
-        self, task: ScheduledTask[Subscription], tick: Tick, timer: Timer
+        self, task: ScheduledTask[Subscription], tick: Tick
     ) -> Future[ClickhouseQueryResult]:
+        timer = Timer("query")
```

Comment:
The name `query` will make this timer share the same name and metric names as the timer that is instantiated for the web interface here: https://github.com/getsentry/snuba/blob/master/snuba/web/views.py#L234

They both share a lot of the same query processing stages from `validate_request_content` onwards through the rest of the stack, so it probably makes sense to keep these stats together so that we can get a big picture overview of all query processing.

The catch is they won't actually be recorded together, since all of the web metrics are prefixed with `snuba.api` and the subscription consumer queries will presumably not be prefixed with that, but `snuba.subscriptions` or something instead.

If we wanted to remove the prefix and record the entrypoint as a tag or something instead, that'd require updating all of the dashboards.